src/wp-admin/includes/nav-menu.php

```diff
@@ -337,6 +337,7 @@ function wp_nav_menu_item_post_type_meta_box( $object, $box ) {
 	global $_nav_menu_placeholder, $nav_menu_selected_id;
 
 	$post_type_name = $box['args']->name;
```

Comment:
After this line seems that
`$post_type      = get_post_type_object( $post_type_name );`
is missing and causing some warnings.

Reply:
It looks like we might need [47209](https://core.trac.wordpress.org/changeset/47209) here.